alpine/resin

```diff
@@ -0,0 +1,66 @@
+#!/sbin/openrc-run
```

Comment:
For example there is a `chdir` directive http://man7.org/linux/man-pages/man8/start-stop-daemon.8.html